### PR TITLE
[doc] One book to rule them all

### PR DESCRIPTION
**Pull request by HU90m**
- Removed the getting started book
- Moved it's content into the main book
- Changed getting started content to use file links where appropriate.

### CODE REVIEW

**Review by jwnrt — APPROVED** [review]
Thanks @HU90m, I think this is for the best.

I could build and browse this locally after re-enabling doxygen and I skimmed around it and it looked fine.

File: SUMMARY.md
Comment (HU90m) [reply SUMMARY.md:9]: Will do

**Comment by hcallahan-lowrisc** [comment]
Grep tells me there is still 19 instances of `https://opentitan.org/guides/getting_started` in the repo, I guess it's probably worth changing them over in this PR too?

**Review by hcallahan-lowrisc — COMMENTED** [review]
Nit: spelling in message of 98445f2a2a13a1aec8f9dad21b405475c6ca7aa7

**Comment by HU90m** [comment]
> Grep tells me there is still 19 instances of `https://opentitan.org/guides/getting_started` in the repo, I guess it's probably worth changing them over in this PR too?

:+1:  Changed

**Comment by HU90m** [comment]
> Does this mean we can (or should) use absolute links now?

Sadly not, because all the documentation is within `/book/` which is not automatically added to absolute links.

**Review by matutem — COMMENTED** [review]
The link changes look good. I will defer to others that have more extensively navigated the new pages for approval.

**Review by marnovandermaas — APPROVED** [review]
LGTM, thanks!

**Comment by marnovandermaas** [comment]
Who still needs to have a look at this before we can merge it?

**Comment by HU90m** [comment]
> Who still needs to have a look at this before we can merge it?

Harry I guess, but I have done all his suggested changes and he's on vacation, so it's good to go in.

**Comment by marnovandermaas** [comment]
Ok, let's merge it and we can always follow up with fixes.

**Pull request merged**
